src/components/PersonInfo.tsx

```diff
@@ -0,0 +1,95 @@
+import React from 'react';
+import {
+  Link,
+  useLocation,
+  useParams,
+  useResolvedPath,
+} from 'react-router-dom';
+import classNames from 'classnames';
+import { Person } from '../types';
+
+type Props = {
+  person: Person;
+  allPeople: Person[];
+};
+
+export const PersonInfo: React.FC<Props> = ({ person, allPeople }) => {
+  const {
+    slug,
+    name,
+    died,
+    born,
+    sex,
+    motherName,
+    fatherName,
+  } = person;
+  const { personId } = useParams();
+  const location = useLocation();
+  const parentPath = useResolvedPath('../').pathname;
+
+  const isSelected = person.slug === personId;
+  const findFather = allPeople.find(human => (
+    human.name === person.fatherName
+  ));
+  const findMother = allPeople.find(human => (
+    human.name === person.motherName
+  ));
+
+  return (
+    <tr
+      data-cy="person"
+      className={classNames({
+        'has-background-warning': isSelected,
+      })}
+    >
+      <td>
+        <Link
+          to={{
+            pathname: parentPath + slug,
+            search: location.search,
+          }}
+          className={classNames({
+            'has-text-danger': sex === 'f',
+          })}
+        >
+          {name}
+        </Link>
```

Comment:
You use Links with the same logic for person link, person's father link and person's mother link. Let's create  component for person link so you just can reuse it